### PR TITLE
feat(console): plain radio

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Implement component: plain radio

## Linear Issue Reference

LOG-2153

## Testing

截屏2022-04-13 下午3 57 31

### CODE REVIEW

**Comment by demonzoo** [comment]
Be aware of the target branch

File: packages/console/src/components/RadioGroup/index.module.scss [hunk 1]
```diff
@@ -1,72 +1,115 @@
...
+
+        &::before {
+          content: '';
+          background: var(--color-on-primary);
+          width: 10px;
+          height: 10px;
+          display: block;
+          border-radius: 50%;
+          border: 2px solid var(--color-on-primary);
+        }
```
Comment (demonzoo) [inline packages/console/src/components/RadioGroup/index.module.scss:101]: Use svg instead?

File: packages/console/src/components/RadioGroup/index.module.scss [hunk 2]
```diff
@@ -1,72 +1,96 @@
...
-        opacity: 100%;
+        opacity: 0%;
+        position: absolute;
+        right: _.unit(5);
+        top: _.unit(5);
+      }
+
+      &.checked {
+        border-color: var(--color-primary);
+        outline: 1px solid var(--color-primary);
```
Comment (demonzoo) [inline packages/console/src/components/RadioGroup/index.module.scss:47]: It's a 2px border in UX design, not 1px border + 1px outline
image

File: packages/core/src/routes/application.ts [hunk 3]
```diff
@@ -50,7 +50,7 @@ export default function applicationRoutes<T extends AuthedRouter>(router: T) {
         type,
         name,
         oidcClientMetadata: buildOidcClientMetadata(oidcClientMetadata),
-        customClientMetadata,
+        customClientMetadata: customClientMetadata ?? {},
```
Comment (demonzoo) [inline packages/core/src/routes/application.ts:53]: Why do we need to fallback undefined to `{}` on server? This should've be done on client side

Comment (wangsijie) [reply packages/core/src/routes/application.ts:53]: this code is for local debug only, added by accident, need to remove.

Comment (demonzoo) [reply packages/core/src/routes/application.ts:53]: ok

File: packages/console/src/components/RadioGroup/index.module.scss (hunk 2 above)
Comment (wangsijie) [reply packages/console/src/components/RadioGroup/index.module.scss:47]: updated, and added hover shadow effect.

File: packages/console/src/components/RadioGroup/Radio.tsx [hunk 4]
```diff
@@ -18,12 +20,13 @@ const Check = () => (
 export type Props = {
   className?: string;
   value: string;
-  title?: string;
+  title?: I18nKey;
```
Comment (demonzoo) [inline packages/console/src/components/RadioGroup/Radio.tsx:23]: AdminConsoleKey?

File: packages/console/src/components/RadioGroup/index.module.scss [hunk 5]
```diff
@@ -1,72 +1,96 @@
...
-        margin-left: _.unit(3);
+      .indicator {
+        border-radius: 50%;
+        border: 2px solid var(--color-neutral-60);
+        display: inline-block;
+        margin-right: _.unit(2);
+
+        &::before {
+          content: '';
+          background: var(--color-on-primary);
```
Comment (demonzoo) [inline packages/console/src/components/RadioGroup/index.module.scss:76]: Use `--color-layer-1` for better dark mode compatibility

File: packages/console/src/components/RadioGroup/index.module.scss [hunk 6]
```diff
@@ -1,72 +1,96 @@
...
+          width: 10px;
+          height: 10px;
+          display: block;
+          border-radius: 50%;
+          border: 2px solid var(--color-on-primary);
```
Comment (demonzoo) [inline packages/console/src/components/RadioGroup/index.module.scss:81]: ditto

**Review by demonzoo — APPROVED** [review]

File: packages/console/src/components/RadioGroup/Radio.tsx (hunk 4 above)
Comment (wangsijie) [reply packages/console/src/components/RadioGroup/Radio.tsx]: updated

File: packages/console/src/components/RadioGroup/index.module.scss (hunk 5 above)
Comment (wangsijie) [reply packages/console/src/components/RadioGroup/index.module.scss]: updated

File: packages/console/src/components/RadioGroup/index.module.scss (hunk 6 above)
Comment (wangsijie) [reply packages/console/src/components/RadioGroup/index.module.scss]: updated

**Pull request merged**